scylla/src/transport/session.rs

```diff
@@ -456,6 +457,71 @@ impl Session {
         response.into_query_result()
     }
 
+    /// Performs many paged queries to collect all results of the given query.  
+    ///
+    /// When receiving large amounts of data it's better to use multiple
+    /// paged queries, a single unpaged query might cause heavy load
+    /// on the database.
+    ///
+    /// `page_size` must be set for the given query.
+    ///
+    /// See [the book](https://cvybhu.github.io/scyllabook/queries/query_all.html) for more information
+    /// # Arguments
+    /// * `query` - query to perform, must have `page_size` set.
+    /// * `values` - values bound to the query, easiest way is to use a tuple of bound values
+    ///
+    /// # Examples
+    /// ```rust
+    /// # extern crate scylla;
```

Comment:
The use of `extern crate` is from Rust 2015, it isn't needed anymore

Reply:
:+1:, removed.